packages/web3-eth-accounts/src/types.ts

```diff
@@ -16,38 +16,8 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { Web3BaseWalletAccount, HexString } from 'web3-types';
-import { FeeMarketEIP1559TxData, AccessListEIP2930TxData, TxData } from './tx/types.js';
 import { AccessListEIP2930Transaction, FeeMarketEIP1559Transaction, Transaction } from './tx';
 
-export type SignatureObject = {
```

Comment:
I'll suggest to move only shared types that are used in 1+ packages in web3-types package, and for moved types re-export those from web3-eth-accounts package for backward compatibility.

Reply:
the problem is that `SignatureObject` is used in web3-eth-accounts but is also part of the `Web3BaseWalletAccount` type in `web3-types`. 
See the type details below:

```ts
export interface Web3BaseWalletAccount {
	// ..
	readonly signTransaction: (tx: Transaction) => Promise<SignTransactionResult>;
	readonly sign: (data: Record<string, unknown> | string) => SignResult;
	//
}
```

Also, none of the `web3-eth-accounts` types are exported from the umbrella `web3` package, but I will reexport these for `web3-eth-accounts` backwards compatibility.